python/cuspatial/cuspatial/tests/test_geoseries.py

```diff
@@ -594,3 +594,33 @@ def test_memory_usage_large():
     geometry = cuspatial.from_geopandas(host_dataframe)["geometry"]
     # the geometry column from naturalearth_lowres is 217kb of coordinates
     assert geometry.memory_usage() == 216793
+
+
+@pytest.mark.parametrize("level", [None, 0, 1])
+@pytest.mark.parametrize("drop", [False, True])
+@pytest.mark.parametrize("inplace", [False, True])
+@pytest.mark.parametrize("name", [None, "ser"])
+def test_reset_index(level, drop, name, inplace):
+    if not drop and inplace:
+        pytest.skip(
+            "For exception checks, see "
+            "test_reset_index_dup_level_name_exceptions"
```

Comment:
Is this just a left over from cudf?

Reply:
It is copied over from cudf, yes. I could rewrite the parameterizations so that this invalid case never occurs. If drop == False and inplace == True for a series `reset_index` an exception is thrown. Since the result has to be a `GeoDataFrame`, inplace is impossible. cudf just built a matrix of parameterizations and skips the impossible ones.

Reply:
I see, if it's expected to fail, instead of skipping, should we just wrap it in `pytest.raises` block?